builtin/providers/aws/resource_aws_route53_zone.go

```diff
@@ -64,6 +86,7 @@ func resourceAwsRoute53ZoneCreate(d *schema.ResourceData, meta interface{}) erro
 	zone := cleanZoneID(*resp.HostedZone.ID)
 	d.Set("zone_id", zone)
 	d.SetId(zone)
+	d.Set("vpc_region", req.VPC.VPCRegion)
```

Comment:
ah yeah. We allow you to omit the `vpc_region` and then you default to using the region of the `aws` provider. In that case I need to make the field a `Calculated: true` field and explicitly set it when omitted. Otherwise I was running into diff mismatch errors during the acceptance tests. I guess this `d.Set` should be up in the `if` block above so it only occurs when using the `vpc` fields. Does that make sense?